Increase max body size and bump template version

madnificent opened the pull request:
Increases the body size of the requests so larger delta messages can be processed.

Co-author: @erikap

madnificent committed:
Increase max body size and bump template version

Increases the body size of the requests so larger delta messages can be
processed.

Co-author: @erikap

sergiofenoll commented:
When booting up a stack locally, this service crashes and due to the `docker-compose.dev.yml` file having `restart: "no"` set, it never reboots.

Logs:
```
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.254372325Z Verifying all current archives from finished jobs
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.257775917Z 
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261055012Z   PREFIX prov: <http://www.w3.org/ns/prov#>
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261085055Z   PREFIX nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#>
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261093987Z   PREFIX nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261100489Z   PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261106161Z 
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261111322Z   SELECT DISTINCT ?job ?physf WHERE {
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261116486Z     GRAPH ?g {
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261122201Z       ?job a <http://mu.semte.ch/vocabularies/ext/FileBundlingJob> ;
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261127863Z           prov:generated ?file ;
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261133023Z           ext:status <http://vocab.deri.ie/cogs#Success> .
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261138668Z       ?physf a nfo:FileDataObject ;
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261143766Z           nie:dataSource ?file .
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261148774Z     }
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261153603Z   }
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.261168678Z Headers set on SPARQL client: {"requestDefaults":{"headers":{"mu-auth-sudo":"true"}}}
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.281512959Z Starting server on 0.0.0.0:80 in production mode
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345828927Z /usr/src/build/node_modules/sparql-client-2/lib/client.js:79
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345870376Z                     throw new Error("Could not connect to SPARQL endpoint.");
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345880712Z                           ^
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345887401Z 
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345892993Z Error: Could not connect to SPARQL endpoint.
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345898331Z     at /usr/src/build/node_modules/sparql-client-2/lib/client.js:79:27
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345903604Z     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345909182Z     at async findAllJobArchives (/usr/src/build/queries/job.js:185:19)
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345914371Z     at async verifyArchiveFiles (/usr/src/build/app.js:104:16)
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345919705Z 
app-kaleidos-file-bundling-1  | 2024-10-03T14:40:43.345924985Z Node.js v20.11.0
```

Not sure why it crashes now and not before, maybe the new mu-auth took a bit longer to boot. Will try to add a retry here. It may not actually be related to this PR.

Only noticed this because  I was missing deltas (due to this service being down).

sergiofenoll committed:
chore: update @lblod/mu-auth-sudo

sergiofenoll committed:
chore: add our own retry mechanism for queries

On boot, sometimes mu-auth isn't ready yet for this service so we should at least retry the query instead of failing.

Maybe there are better ways to have services depend on mu-auth being fully online before starting (some docker depends_on magic perhaps) but that's outside the scope of this branch.
